styles/banner.css

```diff
@@ -0,0 +1,68 @@
+.banner-container {
+    background-image: -moz-linear-gradient( 90deg, rgb(255,255,255) 0%, rgb(243,243,243) 79%, rgb(231,231,231) 100%);
```

Comment:
As per comment for `attention.css`, don't use rgb notation for colors declaration.